plugins/modem-manager/fu-mm-device.c

```diff
@@ -579,6 +699,9 @@ static void
 fu_mm_device_finalize (GObject *object)
 {
 	FuMmDevice *self = FU_MM_DEVICE (object);
+	if (self->attach_idle)
+		g_source_remove (self->attach_idle);
+	g_clear_pointer (&self->qmi_pdc_active_id, g_array_unref);
```

Comment:
I think `g_array_unref(self->qmi_pdc_active_id)` would be much clearer here -- none of the other cleanup funcs are set to null after deref.

Reply:
I wasn't looking for the re-set to NULL after the deref, I was looking for the explicit check of the pointer being non-NULL before doing the deref. I'll change the lines to do the if() check explicitly before the array unref.